rustls-libssl/src/entry.rs

```diff
@@ -447,12 +448,34 @@ entry! {
     }
 }
 
+pub type SSL_CTX_alpn_select_cb_func = Option<
+    unsafe extern "C" fn(
+        ssl: *mut SSL,
+        out: *mut *const c_uchar,
+        outlen: *mut c_uchar,
+        in_: *const c_uchar,
```

Comment:
should this be `_in` instead of `in_`? I'm thrown by the underscore as a suffix instead of a prefix.

Reply:
Oooohhhhhh that makes sense :+1: